### DISCUSSION TITLE
Question 16 [2022]: Sorting out New Zealand [RS]

### DISCUSSION THREAD

**Discussion by magpiedin** [fieldmuseum/EMu-Documentation#171] [Q&A] [answered]
Description: https://fieldmuseum.zendesk.com/agent/tickets/53213

**Comment by magpiedin** [comment]
Response: 
There is a ticket in ZD. Need to tease apart what is a report request to get islands on labels and what is a bigger question of how to correctly document sites from New Zealand. MVK to circle around with Botany to decide what they would like to do collectively, Lauren, Elana and Jamie to circle up with Chris P. to see what the Anthro position is re: NZ and then we will get the Sites Working group together to standardise and document.

Note that If name of country + of island = the same (e.g. Madagascar) and there is an ISO code for it then attach to that record in the Political hierarchy. It is also appropriate to attach to the record in the Island hierarchy  as well

[sgrant to schedule an esites working group]

**Comment by rondlg** [comment, marked as answer]
Meeting scheduled 21st March 2022

esites Working Group Meeting: https://github.com/fieldmuseum/EMu-Documentation/discussions/210
